styx-cli/src/main/java/com/spotify/styx/cli/PlainCliOutput.java

```diff
@@ -143,7 +143,7 @@ public void printWorkflow(Workflow wf, WorkflowState state) {
 
   @Override
   public void printWorkflows(List<Workflow> workflows) {
-    workflows.forEach(wf -> System.out.println(wf.id().toKey()));
+    workflows.forEach(wf -> System.out.println(wf.componentId() + " " + wf.workflowId()));
```

Comment:
This is a breaking change. Better do a code search to understand the impact.

Reply:
I couldn't find anyone that scripts the cli using the `--plain` or `-p` flag. A bunch are doing it without the flag though 😢